cmd/tetra/getevents/getevents.go

```diff
@@ -214,5 +219,6 @@ func New() *cobra.Command {
 	flags.BoolVar(&Options.Timestamps, "timestamps", false, "Include timestamps in compact output")
 	flags.StringVarP(&Options.TTYEncode, "tty-encode", "t", "", "Encode terminal data by file path (all other events will be ignored)")
 	flags.BoolVar(&Options.StackTraces, "stack-traces", true, "Include stack traces in compact output")
+	flags.StringSliceVar(&Options.PolicyNames, "policy-names", nil, "Get events by tracing policy names")
```

Comment:
only nit, maybe this could have been `policies` since it's easier to type than `policy-names` but this is mostly personal preference!

Reply:
No strong opinion from my side on this, LMK if you prefer `policies` over `policy-names`.